CHANGES.rst

```diff
@@ -367,6 +367,8 @@ New Features
 
 - ``astropy.vo``
 
+  - KeyError when converting Table v1.2 numeric arrays fixed. [#4782]
```

Comment:
Correct version section, but this needs to be in "bug fix" sub-section, not "new features".